### DISCUSSION TITLE
Meeting Documentation

### DISCUSSION THREAD

**Discussion by CalvinJDP** [glazet/QuadEVCharger#17] [General]
This post will be edited as we continue to meet.

<html>
<body>
<google-sheets-html-origin>

Date | Time Spent | Present | Action | Goals
-- | -- | -- | -- | --
9/4/23 | 2 hr | Trent, Calvin, Django | Class | N/A
9/5/23 | 2 hr | Trent, Calvin, Django, Neil | First Meeting | N/A
9/6/23 | 1 hr | Trent, Calvin, Django | Research Project | N/A
9/8/23 | 50 min | Trent, Django, Dr. Sudir | Discuss Project | N/A
9/13/23 | 30 min | Trent, Calvin, Django, Dr. Sudir | Discuss Project | N/A
9/13/23 | 1 hr | Trent, Calvin, Django, Dr. Sudir, Bob | Acquired Bob as Advisor | N/A
9/19/23 | 2 hr | Trent, Calvin, Django, Neil | Discussed problem definition and made diagrams | N/A
9/24/23 | 1 hr | Trent, Calvin | Worked on slides and prob definition | N/A
9/26/23 | 2 hr | Trent, Calvin, Django, Neil | Worked on marketing and engineering requirements | N/A
10/03/23 | 1.5 hr | Trent, Calvin, Django, Bob, Dr. Sudir | Discussed current status of project | N/A
10/03/23 | 30 min | Trent, Calvin, Django | Discussed what things to research this week | N/A
10/9/23 | 1 hr | Trent, Calvin, Django | Discussed what things to research upcoming weeks | - find a separate PWM board and ADC board
10/10/23 | 2 hr | Trent, Calvin, Django, Bob, Neil | N/A | N/A
10/13/23 | 4 hr | Trent, Calvin, Django | Worked on Alternate Design Matrices, Discussed Algorithm inputs and outputs | N/A
10/16/23 | 1 hr | Trent, Calvin, Django | Discussed who is doing what for the upcoming week | Talk to advisor about Minimal Viable Product, Testing ADC board, Research and buy Relays
10/17/23 | 2 hr | Trent, Calvin, Django, Bob, Neil, Dr. Sudir | Discussed Minimal Viable product and progress report | Start creating circuit diagrams, testing ADC boards
10/19/23 | 2 hr | Trent, Calvin, Django | Worked on Risk and Challenges, Discussed Algorithm Flow Chart and car priority, Discussed purchasing components | Complete Methodology for Report 
10/23/23 | 1 hr | Trent, Calvin, Django | Worked on Methodology | N/A
10/24/23 | 2 hr | Trent, Calvin, Django, Dr. Salem | Received Feedback on slides from Dr. Salem | Adjust slides based on feedback
10/29/23 | 1 hr | Trent, Calvin, Django | Worked on Gantt Chart and Test Plan Summary | N/A
10/30/23 | 1 hr | Trent, Calvin, Django | Discussed this weeks tasks | N/A
10/31/23 | 2 hr | Trent, Calvin, Django, Neil, Bob | Discussed Open EVSE pilot read and J1772 Negotiation | Testing ESP32-S3 PWM signal, Further work on Gannt Chart, Research charging
11/6/23 | 3 hr | Trent, Calvin, Django | Tested PWM signal | Test Proximity Pilot Circuit
11/7/23 | 2 hr | Trent, Calvin, Django, Neil | Discussed Slides | Adjust slides based on feedback
11/12/23 2 hr | Trent, Calvin | Tested control pilot circuit | Order new parts and re-test
11/13/23 | 2 hr | Trent, Django | Worked on Gantt chart, Troubleshooting Control Pilot circuit | Order new parts and re-test
11/16/23 | 2 hr | Trent, Calvin, Django | Tested control pilot circuit, Tested Microcontroller | Include tests in Slides
11/20/23 | 30 min | Trent, Calvin, Django, Dr. Sudir | Discussed weekly tasks, Briefly discussed presentation expectations | Prepare for practice presentation
11/20/23 | 3 hr | Trent, Calvin, Django | Tested Relay circuit | Prepare for Practice presentation
11/20/23 | 1 hr | Trent, Calvin, Django, Dr. Salem | Class Practice presentation | Adjust slides based on feedback

</body>
</html>

**Comment by djangodemetri** [comment]
10/24/23 NO MEETING SCHEDULED but meeting can still be requested

**Comment by neilh10** [comment]
Well who would request the meeting?       When do you go from "herd mode" to "team mode"
Good to see some detail here and in https://github.com/glazet/QuadEVCharger/discussions/18  
Seems to me the challenge for the team  QuadEVCharger;  there was a discussion Oct 16th (and previous weeks) and then the actions/suggestions from it aren't implemented or considered -  no (attempted) meeting minutes. 

From last meeting my suggestion  was to create two feature issues to track the design and implementation of "PP" and "CP", and then drive the functional decomposition of the features to actual circuits, possible through multiple sub issues, to circuits that can be part of a schematic (kicad?) that can be captured. 
Then this can be integrated with https://github.com/glazet/QuadEVCharger/issues/8 & https://github.com/glazet/QuadEVCharger/issues/10
As an admin item (@glazet ) should give all active team members full control to manage the issues Assignees and labels, and discussions. Settings->Collaborators
